charts/aad-pod-identity/values.yaml

```diff
@@ -14,6 +14,13 @@ image:
 # To match only pods in the namespace containing AzureIdentity set this to true.
 forceNameSpaced: "false"
 
+# When NMI runs on a node where MIC is running, then MIC token request call is also
+# intercepted by NMI. MIC can't get a valid token as to initialize and then
+# assign the identity. Installing an exception for MIC would ensure all token requests
+# for MIC pods directly go to IMDS and not go through the pod-identity validation
+# https://github.com/Azure/aad-pod-identity/blob/master/docs/readmes/README.app-exception.md
+installMICException: "true"
```

Comment:
Is this required by default ? Wouldn't this only be for system assigned identity clusters ?

Reply:
This is required for all helm installs for AKS clusters. In AKS clusters, NMI runs on the same node as MIC and fails to fetch tokens.